edit-post/store/selectors.js

```diff
@@ -96,6 +96,17 @@ export function isEditorSidebarPanelOpened( state, panel ) {
 	return panels ? !! panels[ panel ] : false;
 }
 
+/**
+ * Returns true if a modal is active, or false otherwise.
+ *
+ * @param  {Object}  state Global application state.
+ * @param  {string}  modalName Modal name.
+ * @return {boolean}       Whether the modal is active.
```

Comment:
```
 * @param {Object} state     Global application state.
 * @param {string} modalName Modal name.
 *
 * @return {boolean} Whether the modal is active.
```

Also, `Modal name.` isn't very descriptive. Maybe something like `A string that uniquely identifies the modal.`?